pygotham/about/models.py

```diff
@@ -0,0 +1,29 @@
+"""About models."""
+
+from slugify import slugify
+from sqlalchemy_utils.decorators import generates
+
+from pygotham.core import db
+
+__all__ = ('AboutPage',)
+
+
+class AboutPage(db.Model):
```

Comment:
Those are the types of pages I wanted the copy for. I see it more as "this was the code of conduct for PyGotham 2014" and "this other page is the code of conduct for PyGotham 2015."